src/docfx.website.themes/default/styles/docfx.js

```diff
@@ -500,6 +504,10 @@ $(function () {
           });
           var container = $('#affix > ul');
           container.scrollTop(container.scrollTop() + top - 100);
+
+          // calculate height of #affix
+          var parentHeight = $("#affix").parent().height();
+          $("#affix").css("height", parentHeight - contributionHeight - 50); // minus the height of margin top
         }
```

Comment:
ie looks good while edge not, I'll investigvate which style is not that compatible with Edge

---
In reply to: [75638785](https://github.com/dotnet/docfx/pull/626#discussion_r75638785) [](ancestors = 75638785)